package.json

```diff
@@ -2,203 +2,112 @@
   "name": "snabbdom",
   "version": "2.1.0",
   "description": "A virtual DOM library with focus on simplicity, modularity, powerful features and performance.",
-  "type": "module",
```

Comment:
I'm a little sad to see this. `type: "module"` is the node sanctioned way of declaring this module to be es based. Going back to cjs feels a bit regressive. :(

Reply:
The problem is AFAICT that node will never pick the entry from `module`, always `main`. So if we set `type: module` we would need to throw out cjs completely. And at least I have often issues with testing and ES modules